src/OSPSuite.InstallationValidator.Core/Presentation/MainPresenter.cs

```diff
@@ -50,6 +54,9 @@ public void SelectOutputFolder()
 
       public void Abort()
       {
+         if (!_validationRunning)
```

Comment:
That takes care of the bug you found yesterday where it prompts for cancel when you close the tool even if no validation is running